testsuite/features/support/commonlib.rb

```diff
@@ -32,19 +32,20 @@ def compute_channels_to_leave_running
   # keep the repos needed for the auto-installation tests
   do_not_kill =
     if product == 'Uyuni'
-      CHANNEL_TO_SYNCH_BY_OS_VERSION['15.4']
+      CHANNEL_TO_SYNCH_BY_OS_PRODUCT_VERSION['sles15.4']
     else
-      CHANNEL_TO_SYNCH_BY_OS_VERSION['default']
+      CHANNEL_TO_SYNCH_BY_OS_PRODUCT_VERSION['default']
     end
   [get_target('sle_minion'), get_target('build_host'), get_target('ssh_minion'), get_target('rhlike_minion')].each do |node|
     next unless node
     os_version = node.os_version
     os_family = node.os_family
     next unless ['sles', 'rocky'].include?(os_family)
     os_version = os_version.split('.')[0] if os_family == 'rocky'
-    log 'Can\'t build list of reposyncs to leave running' unless %w[15-SP3 15-SP4 15.4 8].include? os_version
-    do_not_kill += CHANNEL_TO_SYNCH_BY_OS_VERSION[os_version]
+    log 'Can\'t build list of reposyncs to leave running' unless %w[sles15-sp3 sles15-sp4 leap15.4 res8].include? os_version
```

Comment:
```suggestion
    log 'Can\'t build list of reposyncs to leave running' unless %w[sles15-sp3 sles15-sp4 leap15.4-x86_64 res8].include? os_version
```
`leap15.4` does not exist in your changed constants.

Reply:
Good catch!